src/app/pages/event/create.vue

```diff
@@ -1,7 +1,128 @@
 <template>
   <div>
-    <LayoutPageTitle :is-button-event-create-shown="false" :title="title" />
-    <FormEvent v-if="store.jwtDecoded?.role === 'maevsi_account'" />
+    <div v-if="jwtDecoded?.role === 'maevsi_account'">
+      <Stepper
+        v-model="stepIndex"
+        :total-steps="5"
+        class="flex min-h-screen flex-col"
+      >
+        <div class="flex w-full justify-center gap-2 py-4">
+          <StepperItem
+            v-for="step in 5"
+            :key="step"
+            v-slot="{ state }"
+            class="relative flex items-center"
+            :step="step"
+          >
+            <div
+              class="h-2.5 w-2.5 rounded-full border transition-colors duration-200"
+              :class="[
+                state === 'active'
+                  ? 'h-4 w-4 border-accent-weak bg-accent-strong'
+                  : '',
+                state === 'completed'
+                  ? 'h-3 w-3 border-transparent bg-accent-strong'
+                  : '',
+                state !== 'active' && state !== 'completed'
+                  ? 'border-transparent bg-gray-300'
+                  : '',
+              ]"
+            />
+          </StepperItem>
+        </div>
+
+        <div class="flex-1 md:px-12">
+          <h2 class="mb-2 text-2xl font-bold">
+            {{ stepTitles[stepIndex - 1] }}
+          </h2>
+
+          <div class="space-y-6">
+            <EventStepsPrimarySettings
+              v-if="stepIndex === 1"
+              :form="form"
+              :validation="v$"
+              @update-form="
+                (updatedForm: any) => Object.assign(form, updatedForm)
```

Comment:
Uh, let's not use `Object.assign` if possible, I only know it from prototype pollution security vulnerabilities